sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -272,7 +255,7 @@ public boolean isRetryAfter(final @NotNull String itemType) {
     connection.setRequestMethod("POST");
     connection.setDoOutput(true);
 
-    connection.setRequestProperty("Content-Encoding", "UTF-8");
+    connection.setRequestProperty("Content-Encoding", "gzip");
```

Comment:
👍 

"UTF-8" is not a standard value for `Content-Encoding`
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding

And JSON is "by default" encoded in UTF-8 https://tools.ietf.org/html/rfc8259

Normally the character encoding (charset) is part of the `Content-Type` https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

Reply:
that's a very good point, it's been like this since ever though.

Reply:
It didn't hurt, but also didn't do anything useful :)

Reply:
yeah I think this came from sentry-java